source/gui/utilityDialogs.py

```diff
@@ -0,0 +1,58 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2024 NV Access Limited, Luke Davis
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+import wx
+
+from .nvdaControls import MessageDialog
+
+
+class ContinueCancelDialog(MessageDialog):
+	"""
+	This implementation of a `gui.nvdaControls.MessageDialog`, provides `Continue` and `Cancel` buttons as its controls.
+	These serve the same functions as `OK` and `Cancel` in other dialogs, but may be more desirable in some situations.
+	"""
+
+	def __init__(
```

Comment:
@seanbudd I had these double indented as per the codingStandards file, but Pre-commit immediately reduced them to single indenting. Not sure what to do about that, if anything.

Reply:
I think we need to update codingStandards.md